src/DmxInput.h

```diff
@@ -8,12 +8,13 @@
 #ifndef DMX_INPUT_H
 #define DMX_INPUT_H
 
-#ifdef ARDUINO
-#include <dma.h>
-#include <pio.h>
+#if defined(ARDUINO_ARCH_MBED)
+  #include <dma.h>
+  #include <pio.h>
 #else
-#include "hardware/dma.h"
-#include "hardware/pio.h"
+  #include <Arduino.h>
```

Comment:
this looks to me like breaking the build on non-amdroid platforms. if neither ARDUINO_ARCH_MBED. nor ARDUINO are defined, it will try ro inlcude Arduino.h?

Reply:
Thanks for reviewing. Please check the update

```
#if defined(ARDUINO_ARCH_MBED)
  #include <dma.h>
  #include <pio.h>
#else
  #ifdef ARDUINO
    #include <Arduino.h>
  #endif
  #include "hardware/dma.h"
  #include "hardware/pio.h"
#endif
```